File: dotnet/Backend/Models/Quote.cs
namespace Backend.Models;

public class Quote
{
    public int Id { get; set; }
    public int ShowId { get; set; }
    public int CharacterId { get; set; }
    public int EpisodeId { get; set; }
    public string QuoteText { get; set; }
    public bool IsFamous { get; set; }
    public int Likes { get; set; }
    
    // BUG: Inconsistent - some nav properties, some missing
    public Character Character { get; set; }
    // Missing: public Show Show { get; set; }
    // Missing: public Episode Episode { get; set; }
}

File: go/backend/models/quote.go
package models

import "time"

// BUG: Mixed JSON tags and validation
type Quote struct {
	ID          int       `json:"id" gorm:"primaryKey"`
	ShowID      int       `json:"show_id"`
	CharacterID *int      `json:"character_id"`          // BUG: pointer
	EpisodeID   *int      `json:"episode_id"`            // BUG: pointer
	QuoteText   string    `json:"quote_text"`
	Context     string    `json:"context,omitempty"`
	IsFamous    bool      `json:"is_famous"`
	LikesCount  int       `json:"likes_count,omitempty"`
	CreatedAt   time.Time `json:"created_at"`            // BUG: missing omitempty
}

func (Quote) TableName() string {
	return "quotes"
}
